all-the-icons.el

```diff
@@ -120,6 +120,11 @@
   :group 'all-the-icons
   :type 'number)
 
+(defcustom all-the-icons-fonts-subdirectory "all-the-icons/"
```

Comment:
Please don't set a default or the next time a user reinstalls the fonts will wonder where they have gone.

Reply:
If I understand your point here: Removing the default would mean the fonts are not installed in a sub-folder. So users who want the current behaviour of installing all the fonts to the system fonts directory don't have to do anything - because this will be the default behaviour. Additionally, users who want to use a sub-folder can simply do so by setting this variable. 

Is that right? If so, I agree. It's a very nice way of maintaining comptibility with the behaviour users are used to, while adding an additional extension to that behaviour. I will remove the custom value, and wait for your confirmation here to push.

Reply:
that's correct